packages/portfolio-contract/src/type-guards.ts

```diff
@@ -137,6 +137,13 @@ export const BeefyPoolPlaces = {
   },
 } as const satisfies Partial<Record<InstrumentId, PoolPlaceInfo>>;
 
+export const ERC4626PoolPlaces = {
+  ERC4626_vaultU2_Ethereum: {
```

Comment:
```suggestion
  VaultU2: {
```

Protocol and chainName can be found in the object. Or maybe `VaultU2_Ethereum`